docs/SimpleList.md

```diff
@@ -37,19 +37,40 @@ export const PostList = () => (
 
 `<SimpleList>` executes the functions passed as `primaryText`, `secondaryText`, and `tertiaryText` on render, passing the current `record` as parameter. It uses the result to render each List item.
 
-It accepts the following props:
-
-* [`primaryText`](#primarytext) (required)
-* [`secondaryText`](#secondarytext)
-* [`tertiaryText`](#tertiarytext)
-* [`linkType`](#linktype)
-* [`leftAvatar`](#leftavatar)
-* [`leftIcon`](#lefticon)
-* [`rightAvatar`](#rightavatar)
-* [`rightIcon`](#righticon)
-* [`rowStyle`](#rowstyle)
-* [`rowSx`](#rowsx)
-* [`empty`](#empty)
+## Props
+
+| Prop | Required | Type | Default | Description |
+| --- | --- | --- | --- | --- |
+| `primaryText` | Optional | mixed | record representation | The primary text to display. |
```

Comment:
```suggestion
| `primaryText` | Optional | mixed | [record representation](./Resource.html#recordrepresentation) | The primary text to display. |
```

Reply:
We usually don't include links in the props table, which is already hard to read. There should be a link ion the `primaryText` section below, though.